test/e2e/utils_test.go

```diff
@@ -268,6 +296,22 @@ func patchControllerImageFromEnv(t *testing.T, manifestReader io.Reader) (io.Rea
 	return manifestReader, nil
 }
 
+// getKongVersionFromImageTag gets Kong version from tags of Kong image.
+// TODO: what should it do when:
+// - no image provided
+// - image tag could not be parsed into a semver, like "latest", "nightly".
```

Comment:
I believe we can use something like https://github.com/Kong/kubernetes-ingress-controller/blob/665a8e83deb3e1e4e4f41d4a26d6a0635fc6b0b5/test/internal/testenv/testenv.go#L43-L47.

Nevertheless I believe this should land somewhere in https://github.com/Kong/kubernetes-ingress-controller/blob/665a8e83deb3e1e4e4f41d4a26d6a0635fc6b0b5/test/internal/testenv/testenv.go with a proper godoc and be exported.

Reply:
Unfortunately we did not use `testenv` in e2e tests, and e2e uses separate set of envs. I created an issue for this: https://github.com/Kong/kubernetes-ingress-controller/issues/4593.